Example/OriginateUI/GradientViewController.m

```diff
@@ -0,0 +1,177 @@
+//
+//  GradientViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 6/22/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "GradientViewController.h"
+#import "CodeViewController.h"
+#import <QuartzCore/QuartzCore.h>
+@import OriginateUI;
+
+@interface GradientViewController ()
+
+@property UILabel *label;
```

Comment:
All properties should have attributes. E.g. `@property (nonatomic, strong) UILabel *label;`